freebasics/tests/test_views.py

```diff
@@ -0,0 +1,18 @@
+from django.core.urlresolvers import reverse
+from django.test import TestCase, Client
+from django.contrib.auth.models import User
+
+
+class RegistrationViewTest(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+
+    def test_registration_with_password(self):
+        response = self.client.post(reverse('user_register'), {
+            'username': 'testing',
+            'password': 'testing',
+        })
+        self.assertRedirects(response, reverse(
+            'molo.profiles:user_register'))
+        self.assertEquals(User.objects.all().count(), 1)
```

Comment:
@codiebeulaine it's always good to test before and after. So assert there are 0 users before. Also, can you test that the user created is the correct username